com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Artistic/Normal/NormalFromTextureNode.cs

```diff
@@ -159,6 +159,7 @@ internal class NormalFromTextureNode : IStandardNode
                 { "EightSamples", "8 Samples" }
             },
             functionSelectorLabel: "Sample Count",
+            description: "pkg://Documentation~/previews/.mdNormalFromTexture",
```

Comment:
Looks like a copy/paste error is preventing this one from working.

Reply:
Good find! Updated